source/magpie/dataframe.d

```diff
@@ -1375,6 +1385,47 @@ public:
 
         return grp;
     }
+
+    auto asSlice(Type, SliceKind kind)() @property
+    {
+        static if(__traits(isArithmetic, Type))
```

Comment:
do
```
static if(__traits(isArithmetic, Type))
    alias AType = Type;
else 
    alias AType = string;
```

then the two branches are virtually identical.

Reply:
Completely overlooked `Alias` :man_facepalming: 
Addressed in a1b2c3a